lib/src/model/login_option.dart

```diff
@@ -23,6 +23,9 @@ part of flutter_line_sdk;
 
 /// Options related to LINE login process.
 class LoginOption {
+  /// Default request code that LINE login activity will be called with.
+  static const int DEFAULT_CALLBACK_REQUEST_CODE = 8192;
```

Comment:
rename to DEFAULT_ACTIVITY_RESULT_REQUEST_CODE to make it more readable.
In comments, please add 
```dart
  /// Default request code that LINE login activity (in Android Platform) will be called with.
```